### PR TITLE
Tim/bnb vault

### PR DESCRIPTION
**Pull request by timongll**

### CODE REVIEW

File: auctions/binance.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+specVersion: 0.0.2
+schema:
+  file: ./schema.graphql
+dataSources:
+  - kind: ethereum/contract
```
Comment (Sahilgat) [inline auctions/binance.yaml:5]: Think we shouldn't need this, since `GnosisAuction` not used for Binance

File: auctions/binance.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+          handler: handleCancellation
+        - event: ClaimedFromOrder(indexed uint256,indexed uint64,uint96,uint96)
+          handler: handleClaim
+        - event: NewUser(indexed uint64,indexed address)
+          handler: handleNewUser
+  - kind: ethereum/contract
+    name: SwapContract
+    network: bsc
+    source:
+      address: "0xc9008C143Fa07E119F2f114C7F6A04D3982BbC89"
```
Comment (Sahilgat) [inline auctions/binance.yaml:44]: We updated the address, should be: [0xa6438476a0163efd0fff53fd7f5eb17b8b6394c2](https://bscscan.com/address/0xa6438476a0163efd0fff53fd7f5eb17b8b6394c2)

File: auctions/abis/Swap.json
Comment (Sahilgat) [inline auctions/abis/Swap.json:1]: Was this taken from the latest swap contract? Just curious on why there are diffs

File: v2/binance.subgraph.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
+specVersion: 0.0.4
+schema:
+  file: ./schema.graphql
+dataSources:
+  - kind: ethereum/contract
+    name: RibbonsBNBCoveredCall
+    network: bsc
+    source:
+      address: "0xC56d5a5BE96B5fB51C2bA5cBC59AfE77198838F7"
```
Comment (Sahilgat) [inline v2/binance.subgraph.yaml:9]: Address should be: [0x70b9C89FCFeE3310eA6675A707427676FdC2d437](https://bscscan.com/address/0x70b9C89FCFeE3310eA6675A707427676FdC2d437)

File: v2/binance.subgraph.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+          handler: handleDeposit
+        - event: InitiateWithdraw(indexed address,uint256,uint256)
+          handler: handleInitiateWithdraw
+        - event: Withdraw(indexed address,uint256,uint256)
+          handler: handleWithdraw
+        - event: InstantWithdraw(indexed address,uint256,uint256)
+          handler: handleInstantWithdraw
+        - event: Transfer(indexed address,indexed address,uint256)
+          handler: handleTransfer
+        - event: InitiateGnosisAuction(indexed address,indexed address,uint256,indexed address)
```
Comment (Sahilgat) [inline v2/binance.subgraph.yaml:46]: Don't think we need this as well anymore

File: auctions/binance.yaml (hunk 1 above)
Comment (timongll) [reply auctions/binance.yaml:5]: think need to keep it there because functions in the auctions folder uses it

File: auctions/binance.yaml (hunk 2 above)
Comment (timongll) [reply auctions/binance.yaml:44]: updated

File: v2/binance.subgraph.yaml (hunk 3 above)
Comment (timongll) [reply v2/binance.subgraph.yaml:9]: updated

File: auctions/abis/Swap.json
Comment (timongll) [reply auctions/abis/Swap.json:1]: think its just linting

File: v2/binance.subgraph.yaml (hunk 4 above)
Comment (timongll) [reply v2/binance.subgraph.yaml]: updated

File: auctions/binance.yaml (hunk 1 above)
Comment (timongll) [reply auctions/binance.yaml:5]: updated

**Review by Sahilgat — APPROVED** [review]

**Pull request merged**